ScienceDirect.js

```diff
@@ -78,6 +78,15 @@ function getPDFLink(doc, onDone) {
 		return;
 	}
 	
+	// when using a partnered institution network access, construct the the PDF URL directly
+	var pdfURLdirect = doc.querySelector('link[rel="canonical"]').href;
```

Comment:
This can be `attr(doc, 'link[rel="canonical"]', 'href')` — cleaner, and it won't error out if there's no rel="canonical" link on the page.

And can just reuse `pdfURL` here.